Tests/DuplicatedSDKTest/DuplicatedSDKTest/UITest.swift

```diff
@@ -0,0 +1,11 @@
+import XCTest
+
+final class UITestDuplicatedSDK: XCTestCase {
+
+    @MainActor
+    func testExample() throws {
+        let app = XCUIApplication()
+        app.launch()
+        XCTAssertEqual(app.staticTexts["TEST_RESULT"].label, "true")
```

Comment:
`m`: It would be great if the test would fail with something more descriptive than label text isn't true, because when this fails the next time it could be a bit hard to understand what is going on.

Reply:
```suggestion
        XCTAssertEqual(app.staticTexts["TEST_RESULT"].label, "true", "SentrySDK integrations are not being loaded from the same binary. This will lead to undefined behavior.")
```